.appveyor.yml

```diff
@@ -13,6 +13,8 @@ environment:
 
 matrix:
   fast_finish: true
+  allow_failures:
+    - PIP_FLAGS: --pre
```

Comment:
Does this do anything? Or *will* it do anything?

Reply:
Never mind, I just realised it's appveyor, not azure. I forgot we still had this! =P